operator/Dockerfile

```diff
@@ -1,13 +1,9 @@
 # We have to emulate directory layout as in the repo so that imports in go files work fine.
 ARG roxpath=/workspace/src/github.com/stackrox/rox
 
-# Use CentOS Stream 8 image as the base builder because it has recent Go and git packages needed for the build.
-# https://wiki.centos.org/FAQ/CentOSStream#Where_do_I_get_CentOS_Stream_container_images.3F
-FROM quay.io/centos/centos:stream8 as builder-base
-RUN yum update --security -y && yum install -y golang git
+FROM golang:1.17 as builder
```

Comment:
The problem is Go version update. This adds one more place to take care about. Is there any chance you could reuse https://github.com/stackrox/stackrox/blob/master/EXPECTED_GO_VERSION file?

Reply:
How previous `yum install -y golang git` deal with it?
I think github diff is broken as it was merged in #1211

Reply:
Ha-ha, it hasn't.
The fact that https://github.com/stackrox/stackrox/pull/1211 is merged doesn't change that this will be a maintenance problem.

Reply:
Fixed